script.js

```diff
@@ -47,3 +47,199 @@ navLink3.addEventListener('click', () => {
   homePage.style.filter = 'blur(0)';
   logo.style.filter = 'blur(0)';
 });
+
+// popups section
+
+const projectOneCard = {
+  name: 'Tonic',
+  description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry.Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent',
+  featuredImage: 'src="./images/project1.jpg" alt="project1-snapshot"',
+  technologies: ['html', 'css', 'javascript'],
+  liveLink: 'https://aquinasmoriasi.github.io/Personal-portfolio/',
+  sourceLink: 'https://github.com/Aquinasmoriasi/Personal-portfolio',
+};
+
+const projectOneCardDt = {
+  name: 'Tonic',
+  description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum han printer took a galley of type and scrambled it 1960s with the releawn printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. ummy text of the printing and typesetting release of Lorem Ipsum has been the industry\'s  standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea',
+  featuredImage: 'src="./images/pj1-alt.jpg" alt="project4-snapshot"',
+  technologies: ['html', 'css', 'javascript', 'github', 'Ruby', 'Bootstrap'],
+  liveLink: 'https://aquinasmoriasi.github.io/Personal-portfolio/',
+  sourceLink: 'https://github.com/Aquinasmoriasi/Personal-portfolio',
+};
```

Comment:
Every card should have its own popup and every popup should have the content of its card. In your case, you are generating the same popup for all the cards. Consider fixing it.

Reply:
Noted.